composer.json

```diff
@@ -121,6 +118,7 @@
         "psr/log": "^1.0",
         "ramsey/uuid": "^3.8",
         "roave/better-reflection": "^3.5",
+        "sensio/framework-extra-bundle": "^5.2",
```

Comment:
why is this package added, please?

Reply:
It's used in project-base and the dependency was in monorepo installed as a transitive dependency with now removed fos/rest-bundle